README.md

```diff
@@ -68,6 +70,41 @@ export default function App() {
 }
 ```
 
+#### Option 2: Declaring a sketch using abstracted setup and draw functions
```

Comment:
Since this is more of a technique rather than a JavaScript specific thing, perhaps it should be in a separate area since some users might think this isn't possible in typescript for example. I'm not sure what to name the section though, something like "Notes" or "FAQs" (maybe not correct since there would only be this one answer but could be appropriate) or "Techniques"... I don't know, maybe none of those, What do you think?

Reply:
Agreed, let's move this to `Techniques` or something like that if I can find a more appropriate title.

Reply:
Sounds good, let me know what you're thinking and once it's pushed we can move this one forward for the next release :)

Reply:
In general, I moved "Move using abstracted setup and draw functions" to a separate section as part of the "Usage". Also used the same example for JavaScript and TypeScript sketches. @jamesrweb What do you think?

Reply:
Looks okay but why is there a babel configuration there?

Reply:
By mistake - already removed.

Reply:
Looks good to me 👌🏻

Reply:
Wait, the typescript examples are wrong now? The types aren't correctly imported, etc

Reply:
Did you also mean to make all the examples exactly the same?

Reply:
I removed the props from them, but their types are the same.

Reply:
Got it - let me quickly fix the types.

Reply:
@jamesrweb Please verify this again?

Reply:
> Did you also mean to make all the examples exactly the same?

Yes, the idea is to use the same example in JavaScript and TypeScript sketches. Previously, we used a sketch with props in TypeScript. We still have it in the Props section.

Reply:
@jamesrweb I'm done with the changes